domain/relation/state/relation.go

```diff
@@ -207,15 +292,9 @@ WHERE  relation_uuid = $relationUUID.uuid
 	}
 
 	var endpoints []endpoint
-	err = db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
-		err = tx.Query(ctx, stmt, id).GetAll(&endpoints)
-		if errors.Is(err, sqlair.ErrNoRows) {
-			return relationerrors.RelationNotFound
-		}
-		return errors.Capture(err)
-	})
-	if err != nil {
-		return nil, errors.Capture(err)
+	err = tx.Query(ctx, stmt, id).GetAll(&endpoints)
+	if errors.Is(err, sqlair.ErrNoRows) {
+		return nil, relationerrors.RelationNotFound
 	}
 
 	if l := len(endpoints); l > 2 {
```

Comment:
Using single character variables other that the traditional `i` in for loops is difficult to find and read easily.  Using `cnt` or `count` or `length` is easier to see/understand quickly.